src/Symfony/Component/HttpKernel/Tests/DataCollector/LoggerDataCollectorTest.php

```diff
@@ -32,7 +32,7 @@ public function testCollectWithUnexpectedFormat()
 
         $c = new LoggerDataCollector($logger, __DIR__.'/');
         $c->lateCollect();
-        $compilerLogs = $c->getCompilerLogs()->getValue('message');
+        $compilerLogs = $c->getCompilerLogs();
```

Comment:
This looks wrong to me as it means that the value returned by `getCompilerLogs()` changes which would be a BC break.

Reply:
You are right. I've committed some changes to solve this differently.

Reply:
@javiereguiluz Is the profile comparison still up-to-date with the recent changes?